src/js/load-pyodide.js

```diff
@@ -177,7 +182,7 @@ async function _loadPackage(names, messageCallback, errorCallback) {
         continue;
       }
     }
-    let pkgname = Module.packages.orig_case[pkg] || pkg;
+    let pkgname = Module.packages[pkg]?.name || pkg;
```

Comment:
This `?.` works correctly? I've had it cause trouble for me in the past. Maybe that was just an issue with the emscripten code transformation tools that don't understand newer syntaxes.

Reply:
Yes, you are right it's maybe a bit too recent. https://caniuse.com/?search=%3F. So probably better to avoid it.